app/build.gradle

```diff
@@ -55,8 +55,8 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.9.0'
     implementation 'com.google.android.gms:play-services-maps:18.0.2'
     implementation platform('com.google.firebase:firebase-bom:29.1.0')
-    implementation 'com.google.firebase:firebase-auth:21.0.2'
-    implementation 'com.google.firebase:firebase-firestore:24.0.2'
+    implementation 'com.google.firebase:firebase-auth:21.0.3'
```

Comment:
I upgraded each dependency. Do you know if we should keep older versions for some reasons?

Reply:
Ok fine then